extensions/adobe/experience/content-intelligence/webelement.schema.json

```diff
@@ -0,0 +1,125 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/experience/content-intelligence/webelement",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Content Intelligence Web Element",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "description": "Web element metadata",
+  "definitions": {
+    "webelement": {
+      "properties": {
+        "xdm:id": {
+          "title": "Element Id",
+          "type": "string",
+          "meta:immutable": true,
+          "meta:usereditable": false,
+          "description": "The unique ID of the web element."
+        },
+        "xdm:name": {
+          "meta:usereditable": false,
+          "title": "Asset name",
+          "type": "string"
+        },
+        "xdm:path": {
+          "meta:usereditable": false,
+          "title": "Asset path",
+          "type": "string"
+        },
+        "xdm:modifyDate": {
+          "type": "string",
+          "format": "date-time",
+          "meta:usereditable": false,
+          "examples": [
+            "2004-10-23T12:00:00-06:00"
+          ],
+          "description": "The server date and time when the resource was last modified in the repository, such as when a new version of an asset is uploaded or a directory's child resource is added or removed. The date time property should conform to ISO 8601 standard. An example form is \"2004-10-23T12:00:00-06:00\"."
+        },
+        "xdm:type": {
+          "title": "Type",
+          "type": "string",
+          "description": "The type of the element's component.",
+          "meta:usereditable": false
+        },
+        "xdm:title": {
+          "title": "Title",
+          "type": "string",
+          "description": "The title of the web element"
+        },
+        "xdm:template": {
+          "title": "Template",
+          "type": "string",
+          "description": "The path to the element's template"
+        },
+        "xdm:language": {
+          "title": "Language",
+          "type": "string",
+          "description": "The language of of the web element"
+        },
+        "xdm:asset": {
+          "title": "Asset",
+          "description": "The asset referred by the web element.",
+          "$ref": "https://ns.adobe.com/experience/content-intelligence/webasset"
+        },
+        "xdm:linkUrl": {
+          "title": "LinkUrl",
+          "type": "string",
```

Comment:
this should have a validation RegEx.  you can find them throughout XDM

Reply:
doesn't 
```javascript
"format": "uri"
``` 
ensure a validation?

Reply:
Yes it most certainly does.